apps/ui/src/views/SpaceUser/Delegators.vue

```diff
@@ -1,5 +1,35 @@
-<script setup lang="ts"></script>
+<script setup lang="ts">
+import { _n, _p, shorten } from '@/helpers/utils';
+import type { User, Space, SpaceMetadataDelegation } from '@/types';
+import type { Delegate } from '@/composables/useDelegates';
+
+defineProps<{
+  space: Space;
+  delegates: { delegation: SpaceMetadataDelegation; delegate: Delegate }[];
+  user: User;
+}>();
+</script>
 
 <template>
-  <div></div>
+  <div
+    class="bg-skin-bg border-b sticky top-[112px] lg:top-[113px] z-40 flex w-full font-medium space-x-1"
+  >
+    <div class="pl-4 w-[60%] flex truncate">Delegation contract</div>
```

Comment:
What's the purpose of this?

Reply:
Showing the related delegation contract, since you can delegate on multiple contract.

Reply:
I may be missing some context on some discussion you had elsewhere, but judging on the design (https://www.figma.com/design/hmHTau1Y1KYzrBBKLQr5QV/%23design?node-id=5436-5066&t=YDxtUhTDuXjJRPkm-4) the idea is to have a list of individual delegators here, not an aggregation of total delegators/delegatedVP per network/contract.

That said it's true that the design is lacking a solution for when there are multiple delegation contracts in place.